theme_kea/static/description/index.html

```diff
@@ -1,203 +1 @@
-<section class="container">
-    <h2 class="text-muted">Html5 Responsive Bootstrap Theme for Odoo CMS</h2>
```

Comment:
In this commit message, I see `[REM] theme_real_estate: remove outdated theme description` :thinking: